assets/src/components/ViewHeader.js

```diff
@@ -1,43 +1,33 @@
 import React, { useEffect, useState } from 'react'
 import Typography from '@material-ui/core/Typography'
-import useTimeout from '../hooks/useTimeout'
```

Comment:
This is exactly what I was thinking, why do we need a timeout with all that hook fancy code, looks impressive but is that needed really. Thank for keeping it simple

Reply:
Yeah, Gonzalo said the timeout wasn't really necessary, I think the use of `onBlur` makes more sense.